### ISSUE TITLE
Record request specifics

### ISSUE THREAD

**Issue by thenewguy** [ludrao/django-tellme#58]
It would be helpful to have information like submission IP address as well as the currently collected information

**Comment by ludrao** [comment]
Yes though we need to be careful how much data we collect and in anycase let the user know what info we collected. This seems like an improvement on the javascript feedback project  ?
